src/libs/ajax/leonardo/models/app-models.ts

```diff
@@ -15,7 +15,35 @@ export interface AppError extends LeoError {
   traceId?: string
 }
 
-export type AppStatus = 'STATUS_UNSPECIFIED' | 'RUNNING' | 'ERROR' | 'DELETING' | 'DELETED' | 'PROVISIONING' | 'STOPPING' | 'STOPPED' | 'STARTING'
+export type AppStatus = 'STATUS_UNSPECIFIED'
+ | 'RUNNING' | 'ERROR' | 'DELETING' | 'DELETED' | 'PROVISIONING' | 'STOPPING' | 'STOPPED' | 'STARTING'
+
+export type DisplayAppStatus = 'Running'
+    | 'Deleted'
+    | 'Deleting'
+    | 'Creating'
+    | 'Resuming'
+    | 'Status_unspecified'
+    | 'Error'
+    | 'Pausing'
+    | 'Paused'
+
+export interface StatusToDisplay {
```

Comment:
Small nit: Not in love with this name... maybe something like `StatusObj`? Not usually a fan of names with a type in its name, but this really is an internal object to make statuses more handle-able.